### PR TITLE
[blog] Docsy 0.9.0 release report

### PR DESCRIPTION
**Pull request by chalin**
I thought that I'd provide a release report this time around, to offer a bit more context and explanation regarding the changes that have landed. WDYT @LisaFC et al.? I haven't made a final proof-read pass, but I wanted to get this out before (Lisa's) EOD. Feedback is welcome, including whether we should have such a blog post or not.

**Preview**: https://deploy-preview-1854--docsydocs.netlify.app/blog/2024/0.9.0/

### CODE REVIEW

**Comment by chalin** [comment]
FYI, I just added a list of tentative work items for 0.10.0+.

File: userguide/content/en/blog/2024/0.9.0.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+A few more footer improvements for even easier customization are planned
+([#1852]) for the next [major release][#1812].
+
+## Repository Links and other page info {#page-meta}
+
+Getting [repository links] right has eluded Docsy maintainers and contributors
+since 2019 ([#138]). The challenge is ensuring that repository links work for
+all Docsy-based projects regardless of their setup for single- or multi-language
+support or the presence of a homepage or not.
```
Comment (LisaFC) [inline userguide/content/en/blog/2024/0.9.0.md:44]: single- or multi-language
support, or if they have a homepage.

Comment (LisaFC) [inline userguide/content/en/blog/2024/0.9.0.md:44]: single- or multi-language
support, or if they have a homepage.

File: userguide/content/en/blog/2024/0.9.0.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+Repository links grant access to page meta information. Page last-modified ...
+XXXXXXXX
+
+## Look and feel
+
+### Heading self links
+
+Docsy has switched to build-time generation of heading self links using Hugo's
+`render-heading.html` [hook], in favor of client-side rendering via
```
Comment (LisaFC) [inline userguide/content/en/blog/2024/0.9.0.md:72]: replacing client-side rendering via...

File: userguide/content/en/blog/2024/0.9.0.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+`assets/js/anchor.js` &mdash; which has been dropped ([#1460]).
```
Comment (LisaFC) [inline userguide/content/en/blog/2024/0.9.0.md:73]: &mdash; which has been dropped ([#1460]) ->
(dropped in [#1460])

File: userguide/content/en/blog/2024/0.9.0.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+- Otherwise the link is invisible until the user hovers over the heading
+
+### Links are underlined
+
+Docsy now follows recommended **accessibility practice**: page-body **links are
+underlined** by default. For details, see [#1814] and [#1815].
+
+### Bye bye ellipsis
+
+Finally, the [blocks/feature] shortcode no longer includes ellipsis after the
```
Comment (LisaFC) [inline userguide/content/en/blog/2024/0.9.0.md:90]: ellipsis ("...")

File: userguide/content/en/blog/2024/0.9.0.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+- [Footer improvements](#footer)
+- [Repository Links and other page info](#page-meta)
+- [Look and feel](#look-and-feel)
+
+## Footer improvements {#footer}
+
+> Oh my! We've closed [issue #2][#2]!
+
+This release has resolved **_the_ longest standing and first ever issue
+created** over Docsy!
```
Comment (LisaFC) [inline userguide/content/en/blog/2024/0.9.0.md:25]: over Docsy -> in Docsy

**Review by LisaFC — APPROVED** [review]

File: userguide/content/en/blog/2024/0.9.0.md (hunk 5 above)
Comment (chalin) [reply userguide/content/en/blog/2024/0.9.0.md:25]: Fixed

**Comment by chalin** [comment]
Thank you for the feedback @LisaFC. I've implemented all of the changes you suggested.

**Comment by chalin** [comment]
@LisaFC - I've updated the blog post. Feel free to take another look. Otherwise, I'll merge tomorrow.

**Pull request merged**